README.md

```diff
@@ -153,7 +160,11 @@ https://bnc.lt/a/key_live_jbgnjxvlhSb6PGH23BhO4hiflcp3y7ky?data=ExampleBase64Enc
 ##### Functional
 
 **data**  _optional_
-: Default is { }. Base 64 Encoded JSON dictionary of parameters to pass into the app. Default redirects can be overridden with $ios_url, $android_url and $desktop_url. The appearance in social media can be customized with the $og_title, $og_description and $og_image_url keys.
+:  Base 64 Encoded JSON. 
+Create a JSON dictionary of keys and values, eg "{"$og_title":"My App", "$og_description": "This is a great app"}" 
```

Comment:
On this line, let's add the link to custom keys like so:
Create a JSON dictionary of keys and values to modify link behavior with Branch control parameters (see this [link](https://github.com/BranchMetrics/Branch-Public-API#parameters)). You can also put deep link data here if you'd like, eg "{"$og_title":"My App", "$og_description": "This is a great app", "user_id":1245123}"